encoding/preparation.md

```diff
@@ -250,7 +250,83 @@ such as Intel's [SVT-AV1][] will also not be included.
+you may encounter many different audio formats.
+
+On Blu-Rays, 
+you will most likely find audio encoded losslessly, 
+in the form of either DTS-HD Master Audio, Dolby TrueHD, or PCM. 
+DTS-HD MA and Dolby THD are proprietary codecs that use lossless compression, 
+while PCM is simply raw, uncompressed PCM data.
+The usual way handle these formats is to reencode them to other formats - 
+either lossless or lossy, depending on your taste.   
+But first, you need to decode them. 
+The recommended tool for that is ffmpeg, 
+link to which you will find below.
+
+When working with WEB and TV sources, 
+you will most likely have only lossy audio available.
+The most common codecs here are AC-3, E-AC-3 and AAC. 
+Lossily compressed files should generally not be reencoded - 
+the proper way to handle them is to remux (i.e. copy) them to the final file.
+
+### Which codecs to use?
+
+Once you have your lossless files decoded, you need to encode them. 
+Depending on your taste, you may choose a lossy or lossless codec. 
+The two most widely accepted codecs in fansubbing community are FLAC (lossless)
+and AAC (lossy), but recently opus (also lossy) is gaining some popularity, too.
+
+The recommended encoder for FLAC is the official one, 
+and this is the one this guide fill focus on. 
+You can download it [here][FLAC].
+
+The recommended and most widely used AAC encoder is qaac, 
+which you can download [here][qaac], 
+and this is the one this guide will focus on.  
+Nero and Fraunhofer FDK aren't really that much worse, 
+so you can use them if you really want.
+Other AAC encoders are discouraged, 
+since they provide inferior results.
+
+There is also opus, which is gaining some popularity recently. 
+It is currently the most efficient lossy codec, 
+and it's completely FOSS if you're into that. 
+The recommended opus encoder is the official one,
+which you can find [here][opus].
+
+Other codecs are generally not recommended.
+Formats like Monkey's Audio and TAK provide very little gain over FLAC,
+while not being as widely supported,
+and — in the case of TAK —
+closed source. 
+DTS-HD MA and Dolby THD are much less efficient than FLAC,
+and are also closed source.
+MP3 is simply obsolete,
+and Vorbis has been superseded by opus. 
+DTS and AC-3 provide even worse compression than MP3,
+and don't have any reasonable, free encoders.  
+In short - don't bother, unless you really have to, for some reason.
+
+[FLAC]: https://xiph.org/flac/download.html
+[qaac]: https://sites.google.com/site/qaacpage/cabinet
+[opus]: https://opus-codec.org/downloads/
+
+### Lossless or lossy?
+
+This is entirely dependent on you. 
+Some people like the idea of having an (theoretically) perfect copy of the master audio file, 
+don't mind the increase in size, 
+and state that lossless is the only way to go when archiving.
+Others prefer smaller file sizes, 
+knowing that the difference —
+assuming high enough bitrate —
+won't be audible anyway.   
```

Comment:
Another line break occurance.

Reply:
For this example, use commas instead of the em-dash, as the line breaks will make this look ugly.